### ISSUE TITLE
 loss don't declined

### ISSUE THREAD

**Issue by guoyanzi** [sniklaus/pytorch-liteflownet#10]
I want use your code to train myself model，but the loss don't declined，can you help ?

**Comment by sniklaus** [comment]
Thank you for your message and for being interested in my reimplementaiton! I am afraid that this repository focuses on replicating the inference of the official LiteFlowNet implementation and it does not make any efforts to replicate the training. As such, I would suggest you use [the official Caffe version](https://github.com/twhui/LiteFlowNet) for your task. My apologies.
